src/ansys/openapi/common/_util.py

```diff
@@ -365,7 +365,7 @@ def handle_response(response: requests.Response) -> requests.Response:
 
 
 def generate_user_agent(package_name: str, package_version: str) -> str:
-    """Generate a user-agent string in the form <package info> <python info> <os info>.
+    """Generate a user-agent string in the form ``*<package info>*`` ``*<python info>*`` ``*<os info>*``.
```

Comment:
```suggestion
    """Generate a user-agent string in the form *<package info> <python info> <os info>*.
```
@Andy-Grigg I went with the emphasis since it's not a 'literal' string but a variable for a literal string?!

Reply:
I think that makes sense! Lets go with that and see what it comes out like.

Reply:
I tried to commit the change, but it doesn't allow me to do so!

Reply:
I just saw the same thing... I'll do it manually